spec/index.adoc

```diff
@@ -4520,19 +4683,19 @@ The pseudo-code below does _not_ explicitly enforce the restrictions of which im
 
 ....
 ic0.msg_arg_data_size<es>() : i32 =
-  if es.context ∉ {I, U, Q, Ry, F} then Trap {cycles_used = es.cycles_used;}
+  if es.context ∉ {I, U, Q, CQ, Ry, F} then Trap {cycles_used = es.cycles_used;}
```

Comment:
Unrelated, but: I wonder how much we gain from repeating the “context check” in every call in this section, compared to just stating once that “all functions trap if not called in an allowed context according to table XX”

Reply:
We gain a bit more precision.

Reply:
is there really more information there? Or just more precision in the sense that readers are reminded about this check?

Reply:
> Or just more precision in the sense that readers are reminded about this check?

This is what I meant. Or rather that the pseudo-code is more self-contained, i.e., does not rely on additional assumptions outside the actual code.